providers/tutorial/tox.ini

```
@@ -0,0 +1,93 @@
+[tox]
+envlist = py35,py36,py38,py310
+skip_missing_interpreters = true
+skipsdist=True
+
+[testenv]
+allowlist_externals = rm
+commands =
+    {envpython} -m pip -q install ../../checkbox-ng
+    # Required because this provider depends on checkbox-support parsers & scripts
+    {envpython} -m pip -q install ../../checkbox-support
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-resource.provider
+    # Required because this provider depends on the resource provider
+    {envpython} ../resource/manage.py develop
+    {envpython} manage.py validate
+    {envpython} -m coverage run manage.py test
+    {envpython} -m coverage report
+    {envpython} -m coverage xml
+
+[testenv:py35]
+deps =
+    flake8
+    coverage == 5.5
+    distro == 1.0.1
+    Jinja2 == 2.8
+    libsvm == 3.23.0.4
+    MarkupSafe == 0.23
+    natsort == 4.0.3
+    opencv_python == 4.4.0.42
+    psutil == 5.9.5
+    requests == 2.9.1
+    tqdm == 4.19.5
+    urwid == 1.3.1
+    pyparsing == 2.0.3
+    PyYAML == 3.11
+    XlsxWriter == 0.7.3
```

For sure you dont need opencv here. Maybe the ones from the resource provider are enough?

```suggestion
    flake8
    coverage == 5.5
    requests == 2.9.1
    urwid == 1.3.1
    Jinja2 == 2.8
    MarkupSafe == 0.23
    XlsxWriter == 0.7.3
    tqdm == 4.19.5
    pyparsing == 2.0.3
    distro == 1.0.1
    PyYAML == 3.11
```